### PR TITLE
Proposed fix for 446, select notes in piano roll using timeline

### PR DESCRIPTION
**Pull request by curlymorphic**
Proposed fix for #446, select notes in piano roll using timeline, 

matching #821

### CODE REVIEW

File: include/PianoRoll.h [hunk 1]
```diff
@@ -160,6 +160,8 @@ protected slots:
 
 	void hidePattern( Pattern* pattern );
 
+	void selectRegionFromPixels( int x, int y );
```
Comment (Lukas-W) [inline include/PianoRoll.h]: The `(int x, int y)` is confusing me a bit. Shouldn't it be more something like `(int x_start, x_end)`?

**Comment by curlymorphic** [comment]
> The (int x, int y) is confusing me a bit. Shouldn't it be more something like (int x_start, x_end)?

Yes it should. done :)

**Comment by Lukas-W** [comment]
:sparkles:  Perfect, thanks!  :sparkles:

**Comment by curlymorphic** [comment]
:)

**Pull request merged**